build.sh

```diff
@@ -2,67 +2,62 @@
 
 work_dir=`pwd`
 build_tools=$work_dir/.build
-nuget_path=$build_tools/nuget.exe
-configuration="Debug"
 
-artifacts=$work_dir/artifacts
-log_output=$artifacts/logs
+header() {
+    if [ "$TRAVIS" == true ]; then
+        printf "%b\n" "*** $1 ***"
+    else
+        printf "%b\n" "\e[1;32m*** $1 ***\e[0m"
+    fi
+}
 
-. ./scripts/tasks.sh
+header "Cleanup"
+rm -rf artifacts
+mkdir -p $build_tools
 
-#########################
-# Main
+header "Installing dotnet"
+_dotnet_install_script_source="https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0/scripts/obtain/install.sh"
+_dotnet_install_script=$build_tools/install.sh
 
-if [ "$1" == "--quick" ]; then
-    install_dotnet
-    publish "OmniSharp" || { echo >&2 "Failed to quick build. Try to build the OmniSharp without --quick switch."; exit 1; }
-    exit 0
-fi
+curl -o $_dotnet_install_script $_dotnet_install_script_source -s
+chmod +x $_dotnet_install_script
 
-if [ "$1" == "--install" ]; then
-    install_dotnet
-    if [ -d ~/.omnisharp/local ]; then
-        echo "Removing local omnisharp ..."
-        rm -rf ~/.omnisharp/local
-    fi
+if [ "$TRAVIS" == true ]; then
+    echo "Installing dotnet from beta channel for CI environment ..."
 
-    mkdir -p ~/.omnisharp/local
+    $_dotnet_install_script -c beta -d "$work_dir/.dotnet"
+    dotnet="$work_dir/.dotnet/bin/dotnet"
```

Comment:
I see. 

I'll remove the `.dotnet` specific thing. As soon as `dotnet-cli` comes up with a proper version management we can begin adopt that solution. Currently, since we stay on the current edge build, there isn't a reason to trade off the simplicity.